uwblueprint/jump-math
# feat: add view all states to teacher dashboard

## Notion ticket link

[Update View More in Teacher Dashboard](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)


## Implementation description
* Update "View all" link in teacher dashboard to match new designs 
<img width="1436" alt="image" src="https://github.com/uwblueprint/jump-math/assets/69697180/f215ffcf-021c-4c6b-85a6-89f5d0048626">


## Steps to test
1. Verify that the buttons link to the correct pages
2. Verify that the UI matches the mocks
<img width="734" alt="image" src="https://github.com/uwblueprint/jump-math/assets/69697180/4061a5b1-c8e0-43b1-8dd7-e86dbcd74400">


## What should reviewers focus on?
* 


## Checklist
